x-pack/plugins/security_solution/public/explore/components/risk_score/top_risk_score_contributors_alerts/index.tsx

```diff
@@ -0,0 +1,133 @@
+import { RiskScoreEntity } from '../../../../../common/risk_engine';
+import type { HostRiskScore, UserRiskScore } from '../../../../../common/search_strategy';
+import { ALERTS_TABLE_REGISTRY_CONFIG_IDS } from '../../../../../common/constants';
+import { AlertsTableComponent } from '../../../../detections/components/alerts_table';
+import { GroupedAlertsTable } from '../../../../detections/components/alerts_table/alerts_grouping';
+import { useGlobalTime } from '../../../../common/containers/use_global_time';
+import { useDeepEqualSelector } from '../../../../common/hooks/use_selector';
+import { inputsSelectors } from '../../../../common/store/inputs';
+import { useUserData } from '../../../../detections/components/user_info';
+import { useSourcererDataView } from '../../../../common/containers/sourcerer';
+import { SourcererScopeName } from '../../../../common/store/sourcerer/model';
+
+export interface TopRiskScoreContributorsAlertsProps {
+  toggleStatus: boolean;
+  toggleQuery?: (status: boolean) => void;
+  riskScore: HostRiskScore | UserRiskScore;
+  riskEntity: RiskScoreEntity;
+  loading: boolean;
+}
+
+export const TopRiskScoreContributorsAlerts: React.FC<TopRiskScoreContributorsAlertsProps> = ({
+  toggleStatus,
+  toggleQuery,
+  riskScore,
+  riskEntity,
+  loading,
+}) => {
+  const { to, from } = useGlobalTime();
+  const [{ loading: userInfoLoading, signalIndexName, hasIndexWrite, hasIndexMaintenance }] =
+    useUserData();
+  const { runtimeMappings } = useSourcererDataView(SourcererScopeName.detections);
+  const getGlobalFiltersQuerySelector = useMemo(
+    () => inputsSelectors.globalFiltersQuerySelector(),
+    []
+  );
+  const getGlobalQuerySelector = useMemo(() => inputsSelectors.globalQuerySelector(), []);
+
+  const query = useDeepEqualSelector(getGlobalQuerySelector);
+  const filters = useDeepEqualSelector(getGlobalFiltersQuerySelector);
+
+  const inputFilters = useMemo(() => {
+    const riskScoreEntity =
+      riskEntity === RiskScoreEntity.host
+        ? (riskScore as HostRiskScore).host
+        : (riskScore as UserRiskScore).user;
+
+    const riskInputs = (riskScoreEntity?.risk?.inputs ?? []) as RiskInputs;
+    return [
+      {
+        meta: {
+          alias: null,
+          negate: false,
+          disabled: false,
+        },
+        query: {
+          terms: {
+            _id: riskInputs.map((item) => item.id),
+          },
+        },
+      },
+    ];
+  }, [riskScore, riskEntity]);
+
+  const renderGroupedAlertTable = useCallback(
+    (groupingFilters: Filter[]) => {
+      return (
+        <AlertsTableComponent
+          configId={ALERTS_TABLE_REGISTRY_CONFIG_IDS.RISK_INPUTS}
+          flyoutSize="m"
+          inputFilters={[...inputFilters, ...filters, ...groupingFilters]}
+          tableId={TableId.alertsRiskInputs}
+        />
+      );
+    },
+    [inputFilters, filters]
+  );
+
+  return (
+    <EuiPanel hasBorder data-test-subj="topRiskScoreContributorsAlerts">
```

Comment:
fyi, I'm not going to block this going forward, but I did want you to know that there are some known performance items with the alert table, that users of the risk score _may_ run into and you should be aware of https://github.com/elastic/security-team/issues/7217